### PR TITLE
doc(dbt): social media roll-up README edit for Fivetran style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
Closes https://fivetran.height.app/T-161520

Edits social media roll-up README for style and clarity. Please check my edits for technical accuracy. I've also left a question inline.

### CODE REVIEW

**Comment by fivetran-joemarkiewicz** [comment]
Thanks so much @fivetran-chloe for reviewing! These look great and will merge 😄

**Comment by fivetran-chloe** [comment]
Great, thanks!

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ packages:
 ```
 
 ## Package Maintenance
-The Fivetran team maintaining this package **only** maintains the latest version. We highly recommend you keep your `packages.yml` updated with the [dbt hub latest version](https://hub.getdbt.com/fivetran/social_media_reporting/latest/). You may refer to the [CHANGELOG](/CHANGELOG.md) and release notes for more information on changes across versions.
+The Fivetran team maintaining this package **only** maintains the latest version. We highly recommend that you keep your `packages.yml` file updated with the [dbt hub's latest version](https://hub.getdbt.com/fivetran/social_media_reporting/latest/). Read the [CHANGELOG](/CHANGELOG.md) and release notes for more information on changes across versions.
```
Comment (fivetran-chloe) [inline README.md:33]: We mean the latest version in the dbt hub, correct?

Comment (fivetran-joemarkiewicz) [reply README.md:33]: That is correct. Should we explicitly identify the dbt hub here?

Comment (fivetran-chloe) [reply README.md:33]: I may just reword it to make that clear

Comment (fivetran-chloe) [inline README.md:33]:
```suggestion
The Fivetran team maintaining this package **only** maintains the latest version. We highly recommend that you keep your `packages.yml` file updated with the [latest version in the dbt hub](https://hub.getdbt.com/fivetran/social_media_reporting/latest/). Read the [CHANGELOG](/CHANGELOG.md) and release notes for more information on changes across versions.
```

Comment (fivetran-chloe) [reply README.md]: Ok, made the edit - what do you think?

Comment (fivetran-joemarkiewicz) [reply README.md]: That is great, thanks!

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM thanks so much @fivetran-chloe! 🙌
